ElectronClient/gui/ConfigScreen.jsx

```diff
@@ -316,6 +347,15 @@ class ConfigScreenComponent extends React.Component {
 						style={selectStyle}
 						onChange={event => {
 							updateSettingValue(key, event.target.value);
+							// if Auto Detect option is ON theme should change according to preferred theme options
+							if (settings['themeAutoDetect']) {
```

Comment:
I'll remove this if() statement as it's not needed now. It was there because of previous spec in which `Preferred Light/Dark` options were always visible